package.json

```diff
@@ -9,12 +9,12 @@
   "source": "src/index.ts",
   "exports": {
     "require": "./dist/index.cjs",
-    "default": "./dist/index.modern.js"
+    "default": "./dist/index.modern.js",
+    "types": "./dist/index.d.ts"
   },
   "main": "./dist/index.cjs",
   "module": "./dist/index.module.js",
   "unpkg": "./dist/index.umd.js",
-  "types": "./dist/index.d.ts",
```

Comment:
According to https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html it very much does belong here. Can you explain why it doesn't?

In short, I'm worried that there are setups that expect it to be in `types` and won't look in `export.types`. Can you explain more about what all this means?